test/functional/wallet_resendwallettransactions.py

```diff
@@ -64,6 +64,10 @@ def run_test(self):
         # Transaction should be rebroadcast approximately 24 hours in the future,
         # but can range from 12-36. So bump 36 hours to be sure.
         node.setmocktime(now + 36 * 60 * 60)
+        # Give some time for trickle to occur
+        time.sleep(1)
+        node.syncwithvalidationinterfacequeue()
```

Comment:
Can you explain why this is needed, what this is trying to achieve? #19893

Reply:
I copied it from elsewhere; it works fine without the `syncwithvalidationinterfacequeue` call, so I've removed it.